core/src/main/scala/org/apache/spark/metrics/sink/GraphiteSink.scala

```diff
@@ -50,30 +52,30 @@ private[spark] class GraphiteSink(val property: Properties, val registry: Metric
     throw new Exception("Graphite sink requires 'port' property.")
   }
 
-  val host = propertyToOption(GRAPHITE_KEY_HOST).get
-  val port = propertyToOption(GRAPHITE_KEY_PORT).get.toInt
+  private val host = propertyToOption(GRAPHITE_KEY_HOST).get
+  private val port = propertyToOption(GRAPHITE_KEY_PORT).get.toInt
 
-  val pollPeriod = propertyToOption(GRAPHITE_KEY_PERIOD) match {
+  private val pollPeriod = propertyToOption(GRAPHITE_KEY_PERIOD) match {
     case Some(s) => s.toInt
     case None => GRAPHITE_DEFAULT_PERIOD
   }
 
-  val pollUnit: TimeUnit = propertyToOption(GRAPHITE_KEY_UNIT) match {
-    case Some(s) => TimeUnit.valueOf(s.toUpperCase(Locale.ROOT))
+  private val pollUnit: TimeUnit = propertyToOption(GRAPHITE_KEY_UNIT) match {
+    case Some(s) => TimeUnit.valueOf(s.toUpperCase())
```

Comment:
Hi, @jerryshao .
Is there any reason to remove Locale.ROOT?
This is added by SPARK-20156 to fix local bug.